docs/howto/static-files/deployment.txt

```diff
@@ -106,7 +106,7 @@ Here's how this might look in a fabfile::
         project.rsync_project(
             remote_dir = env.remote_static_root,
```

Comment:
You could remove the spaces are the equal signs while you're here as that's a flake8 violation.